saleor/static/js/components/product-filters.js

```diff
@@ -0,0 +1,27 @@
+export default $(document).ready((e) => {
+  $('.toggle-filter').each(function () {
+    let icon = $(this).find('.collapse-filters-icon');
+    let ele = $(this).find('.filter-form-field');
```

Comment:
`ele` also isn't the best name. Names like `e` or `el` are okay in anonymous functions in obvious context, here the name should definitely refer to the DOM element it holds.